fanuc_lrmate200id7l_moveit_config/launch/trajectory_execution.launch.xml

```diff
@@ -7,7 +7,7 @@
   <param name="moveit_manage_controllers" value="$(arg moveit_manage_controllers)"/>
 
   <!-- When determining the expected duration of a trajectory, this multiplicative factor is applied to get the allowed duration of execution -->
-  <param name="trajectory_execution/allowed_execution_duration_scaling" value="1.2"/> <!-- default 1.2 -->
+  <param name="trajectory_execution/allowed_execution_duration_scaling" value="10.0"/> <!-- default 1.2 -->
```

Comment:
If you'd like your MoveIt package to be included in this repository, then please don't change the default scaling values.

The MoveIt packages provided here are a convenience in any case (so 'real' users are expected to create their own, based on their robot setup and workcell) and any scaling is only needed because of the way `fanuc_driver` and `fanuc_driver_exp` work.

A more performant driver won't need this scaling and that would make the trajectory execution tracking feature not work correctly with those drivers.

Reply:
ugh, I forget to push my specific changes to another branch, the HEAD is still at feature branch, Anyway I will omit latest commit

Reply:
@jediofgever: have you updated the PR to not include the changes to the duration scaling?

Reply:
sorry for late reply @gavanderhoorn, yes I have updated them to be default, in the branch they are already default values now.

Reply:
bd94e3096fd26e8a1cd56219c0ade258fc46369b appears to be the last commit in the PR. That seems to still use the changed values.

Reply:
I had a hard time to understand why this commit appears latest in the PR but not in the branch
![ss](https://user-images.githubusercontent.com/18460336/66573983-102d1800-eba6-11e9-9767-05597e5b959b.png)

Reply:
I don't know either.

Reply:
maybe i can close this PR and create new one if that is necessary, in the feature branch the variables are set to default value

Reply:
i did one more commit, https://github.com/ros-industrial/fanuc_experimental/pull/48/commits/423e9683e9d16a8c078275ebc415eee5afaba2c7
just added a space